desc/backend.py

```diff
@@ -43,40 +33,38 @@
                     + "installed JAX with GPU support?"
                 )
                 set_device("cpu")
-            x = jnp.linspace(0, 5)
-            y = jnp.exp(x)
         use_jax = True
-        if verbose:
-            print(
-                f"DESC version {desc.__version__}, "
-                + f"using JAX backend, jax version={jax.__version__}, "
-                + f"jaxlib version={jaxlib.__version__}, dtype={y.dtype}"
-            )
-        del x, y
     except ModuleNotFoundError:
         jnp = np
-        x = jnp.linspace(0, 5)
-        y = jnp.exp(x)
         use_jax = False
         set_device(kind="cpu")
         warnings.warn(colored("Failed to load JAX", "red"))
+
+
+def print_info():
```

Comment:
`print_environment_info` or `print_backend_info` might be more descriptive.

Also should mention this in the docs, maybe along with `set_device` and mention it in the changelog that it will no longer print info by default.

Should also call this in `__main__` so that when running a simple command line input its the same